File: Camera/app/src/main/java/com/example/camera/MainActivity.java
package com.example.camera;

import androidx.appcompat.app.AppCompatActivity;

import android.app.Activity;
import android.content.Intent;
import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.net.Uri;
import android.os.Bundle;
import android.os.Environment;
import android.provider.MediaStore;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

import java.io.File;

public class MainActivity extends AppCompatActivity {
    ImageView iv_camera;
    Button btn_camera;
    File file;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        File sdcard = Environment.getExternalStorageDirectory();
        file = new File(sdcard, "capture.jpg");

        iv_camera = findViewById(R.id.iv_camera);
        btn_camera = findViewById(R.id.btn_camera);

        btn_camera.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                capture();
            }
        });
    }

    public void capture() {
        Intent intent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);

        intent.putExtra(MediaStore.EXTRA_OUTPUT, Uri.fromFile(file));

        startActivityForResult(intent, 101);
    }

    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);

        if (requestCode == 101 && resultCode == Activity.RESULT_OK) {
            BitmapFactory.Options options = new BitmapFactory.Options();

            options.inSampleSize = 8;
            Bitmap bitmap = BitmapFactory.decodeFile(file.getAbsolutePath(), options);

            iv_camera.setImageBitmap(bitmap);
        }
    }
}

File: AndroidLab/app/src/main/java/com/example/androidlab/MainActivity.kt
package com.example.androidlab

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

// AppCompatActivity를 상속받아 MainActivity 클래스를 정의
// AppCompatActivity는 Activity의 하위 클래스이므로 MainActivity는 액티비티 컴포넌트 클래스
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // setContentView() 함수 : 매개변수에 지정한 내용을 액티비티 화면에 출력 (activity_main.xml 파일의 내용을 화면에 출력)
        setContentView(R.layout.activity_main)
    }
}